interface/forms/clinical_notes/templates/report.html.twig

```diff
@@ -35,7 +35,7 @@ The Report template of the clinical note form.
                 </div>
                 {% endif %}
             </header>
-            <section>{{ n.description|text|nl2br }}</section>
+            <section class="mb-3">{{ n.description|text|nl2br }}</section>
```

Comment:
@robertdown , not sure if this is best place to increase the bottom margin of each separate note (without separation, it made it more difficult to read them since they were all squashed together)